general/3_git2/Ann Horlacheva_Git2/task 2

```diff
@@ -0,0 +1,14 @@
+Merge vs Rebase
+
+Integrate changes using both
+
+`git merge`
```

Comment:
Let's start from the good part: I'm happy with you using back-ticks. I hope that you intended to get code rendering such as `this`.